tests/integration-tests/tests/common/scaling_common.py

```diff
@@ -156,3 +160,14 @@ def get_batch_ce_min_size(stack_name, region):
         .get("computeResources")
         .get("minvCpus")
     )
+
+
+def emulate_ice_in_cluster(cluster):
+    """
+    Includes an override file that emulates an ICE error in a cluster.
+
+    It applies the override patch to launch ice for nodes in
+    "ice-compute-resource" and "ice-cr-multiple" compute resources with an ICE error
```

Comment:
NIT is it possible to have these names as input? So we don't have to remember for which queue the ice is emulated, and build the test config accordingly, but it's the other way around, i.e. the test config is built and than the override is setup based on the name of the queue set for the test

Reply:
It should be possible to pass the target queue names as parameters for the overrides script. I'll run some tests and confirm.